exporter/elasticsearchexporter/integrationtest/exporter_bench_test.go

```diff
@@ -79,6 +81,35 @@ func benchmarkLogs(b *testing.B, batchSize int, mappingMode string) {
 	require.NoError(b, exporter.Shutdown(ctx))
 }
 
+func benchmarkMetrics(b *testing.B, batchSize int, mappingMode string) {
```

Comment:
I wonder if benchmark results would make sense to also be included in the component's docs. 

Like what we have at https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/9fbf26df04b81397a9091b6cfdb7583455ce684e/examples/kubernetes/README.md?plain=1#L48.

Reply:
Let's do this in a follow up. We'll need to think about the value of publishing throughput numbers, as these benchmarks in integrationtest uses a stubbed Elasticsearch instead of a real one. At the moment these benchmarks are useful to compare perf across mapping modes and signal types to identify bottlenecks.

Reply:
+1 on @carsonip 's comment, the purpose of current benchmarks is to spot errors across PRs with no standardization on machine types OR Elasticsearch backend used so not much useful to document this at the current point. It could be used to plot the performance of the exporter but we would probably need some standardization and automation to make this useful.